packages/serverpod_client/lib/src/serverpod_client_exception.dart

```diff
@@ -1,4 +1,4 @@
-import 'dart:html' if (dart.library.io) 'dart:io';
+import './_sdk/html.dart' if (dart.library.io) './_sdk/io.dart';
```

Comment:
Suggestion: We are only interested in the class `HttpStatus` from these packages. It would be nice only to expose this class to these imports. 

I.E. 

```dart
export 'dart:io' show HttpStatus;
```

```dart
export 'dart:html' show HttpStatus;
```

And rename the files to reflect that we are only taking the `HttpStatus` class.